duckdb-vortex/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.22)
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
This is also set in https://github.com/spiraldb/duckdb/blob/main/CMakeLists.txt?plain=1#L29. But doesn't hurt to also set this explicitly in our top-level cmake file.